en/developer_guide/specific_backbone_view.rst

```diff
@@ -0,0 +1,40 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+As all the Backoffice of Open Orchestra is based on Backbone view system, all the specific event
+linked to a view are linked to the view directly.
+
+In some cases, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page will not suit your needs. There are some new behaviour to add to the form, such as a
+specific event when you edit a field.
```

Comment:
Maybe you could rephrase the intro sentence because it says something redundant : "the specific event linked to a view are linked to the view" which won't be helpfull for someone discovering Open Orchestra :)

Maybe that intro could tell that an edit form is composed of two parts, the backbone view and the edit form template, and that the form events should be bind in the backbone view, not the template form.